spec/services/networkedgev1/oas3.fetched/ne-v1-catalog-ne_v1.yaml

```diff
@@ -0,0 +1,7245 @@
+      pageNumber:
+        type: integer
+        example: 1
+      content:
+        type: array
+        items:
+          $ref: '#/definitions/GETConnectionByUuidResponse'
+  GetServProfServicesRespContent:
+    type: object
+    properties:
+      uuid:
+        type: string
+        example: xxxxx191-xx70-xxxx-xx04-xxxxxxxa37xx
+      name:
+        type: string
+        example: test
+      authKeyLabel:
+        type: string
+        example: Authorization Key
+      connectionNameLabel:
+        type: string
+        example: Connection Name
+      requiredRedundancy:
+        type: boolean
+        example: false
+      allowCustomSpeed:
+        type: boolean
+        example: false
+      speedBands:
+        type: array
+        items:
+          $ref: '#/definitions/SpeedBand'
+      metros:
+        $ref: '#/definitions/GetServProfServicesRespContentMetros'
+      createdDate:
+        type: string
+        example: '2018-03-22T04:34:48.231Z'
+      createdBy:
+        type: string
+        example: Sandbox User
+      lastUpdatedDate:
+        type: string
+        example: '2018-04-03T00:30:57.055Z'
+      lastUpdatedBy:
+        type: string
+        example: Sandbox User
+      vlanSameAsPrimary:
+        type: boolean
+        example: false
+      tagType:
+        type: string
+        example: CTAGED
+      ctagLabel:
+        type: string
+        example: Seller-Side C-Tag
+      apiAvailable:
+        type: boolean
+        example: false
+      selfProfile:
+        type: boolean
+        example: false
+      profileEncapsulation:
+        type: string
+        example: Dot1q
+      authorizationKey:
+        type: string
+        example: '535235'
+      organizationName:
+        type: string
+        example: Equinix-ADMIN
+      private:
+        type: boolean
+        example: false
+      features:
+        $ref: '#/definitions/GetServProfServicesRespContentfeatures'
+  SpeedBand:
+    type: object
+    properties:
+      speed:
+        type: number
+        format: double
+        example: 50
+      unit:
+        type: string
+        example: MB
+  GetServProfServicesRespContentMetros:
+    type: object
+    properties:
+      code:
+        type: string
+        example: SV
+      name:
+        type: string
+        example: Silicon Valley
+      ibxs:
+        type: array
+        items:
+          type: string
+          example: SV1
+      inTrail:
+        type: boolean
+        example: false
+      displayName:
+        type: string
+        example: Silicon Valley
+  GetServProfServicesRespContentfeatures:
+    type: object
+    properties:
+      cloudReach:
+        type: boolean
+        example: true
+      testProfile:
+        type: boolean
+        example: false
+  PatchRequest:
+    type: object
+    properties:
+      accessKey:
+        type: string
+        example: AKIAIOSFODNN7EXAMPLE 
+      secretKey:
+        type: string
+        example: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
**NOTE:** I had to change these example values in the fetched _and_ patched spec because the examples in the original spec were being flagged as secrets.  I confirmed that the example values in the original spec are _not_ a valid or active access key, but it's better to use example values that are _clearly_ examples.  The example values used here were copied from [the AWS IAM docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).